fem/fe.cpp

```diff
@@ -13,7 +13,6 @@
 
 #include "fe.hpp"
 #include "fe_coll.hpp"
-#include "../mesh/nurbs.hpp"
 #include "bilininteg.hpp"
 #include <cmath>
```

Comment:
Are these includes (besides fe.hpp) needed here?

Reply:
These were carried over from the original fe.cpp but you make a good point. I'll double check them and make sure they're still needed here. I may even add comments explaining why they're needed if it isn't obvious...

Reply:
Apparently they were not needed. I have removed them. Thanks for catching this.